ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -1048,10 +1052,6 @@ keywordsCanBeId
    | LAST
    | LIST
    | VALUES
-   | EARLIEST
-   | EARLIEST_TIME
-   | LATEST
-   | LATEST_TIME
```

Comment:
why remove this?

Reply:
Those keywords are not present in lexer.g4, another words, they are invalid here.